### PR TITLE
Updated readme 

### PR DESCRIPTION
**Pull request by harsh4786**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
 # wordcel
```
Comment (KunalBagaria) [inline README.md:1]: Unrelated to the PR, but this should be `Wordcel`

**Review by KunalBagaria — CHANGES_REQUESTED** [review]
lgtm, please update the title's capitalization as well

File: README.md (hunk 1 above)
Comment (harsh4786) [reply README.md]: done

**Review by KunalBagaria — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 
-Core Protocol
+Core Protocol, comprising of on-chain programs, IDL and tests.
+## Account model 
+ ![worcel accounts model](https://github.com/harsh4786/wordcel/blob/master/wordcel_accounts.png)
+  
 
-
-## Users can
+## The program is modular and can be used to create generic content not limited to written content.
```
Comment (abishekk92) [inline README.md:9]: Can you talk about random_hash in this and why it is required?

**Comment by abishekk92** [comment]
Left my comment, otherwise this PR looks good. Can merge it after the requested changes are added.

File: README.md (hunk 2 above)
Comment (harsh4786) [reply README.md:9]: sure, just give me a moment

Comment (harsh4786) [reply README.md:9]: done!

Comment (harsh4786) [reply README.md:9]: lmk if this works

**Review by abishekk92 — APPROVED** [review]

**Comment by abishekk92** [comment]
Merged thanks for your contribution @harsh4786

**Pull request merged**